Exam/CT/src/exercice2/Launcher.java

```diff
@@ -0,0 +1,22 @@
+package exercice2;
+
+import java.awt.image.*;
+import javax.swing.*;
+
+public class Launcher {
+
+		public static void main(String[]args)throws Exception
+		
+		{
+			//on charge lenna
+			int Lenna[][] = Util.getLennaGreenBuffer();
+			//on utilise notre fonction qui a deja de probleme de code
+			IImageBuffer masking = new MaskedImage(Lenna);
```

Comment:
This line does not compile...yet. And this exactly the problem you were asked to solve.
What do you need to make `MaskedImage` **compatible** with `IImageBuffer`?